comp/core/workloadmeta/collectors/internal/process/process_collector_nop.go

```diff
@@ -16,3 +16,9 @@ import (
 func GetFxOptions() fx.Option {
 	return nil
 }
+
+// LocalProcessCollectorIsEnabled returns whether the local process collector is enabled
+// based on agent flavor and config values. Always returns false outside of Linux.
+func LocalProcessCollectorIsEnabled() bool {
+	return false
+}
```

Comment:
Do we still needs this function after moving into the util package?

Reply:
Ah, I missed that. Thanks